api/database/factories/PoolCandidateSearchRequestFactory.php

```diff
@@ -49,6 +50,7 @@ public function definition()
             'position_type' => $this->faker->randomElement(PoolCandidateSearchPositionType::cases())->name,
             'reason' => $this->faker->randomElement(PoolCandidateSearchRequestReason::cases())->name,
             'community_id' => $community->id,
+            'user_id' => $user->id,
```

Comment:
```suggestion
            'user_id' => $user?->id,
```

Reply:
This would probably fix the currently failed CI test.

Reply:
Awesome thanks :+1: Fixed in [bb360d9](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11275/commits/bb360d9a772ebe3a4d79357f56c91a947f4a8127)